lib/post.php

```diff
@@ -108,6 +125,28 @@ public function type() {
 	}
 
 	/**
+	 * Is the post type blacklisted?
+	 *
+	 * If the $page parameter is specified, this function will additionally
+	 * check if the query is for one of the pages specified.
+	 *
+	 * @param string $post_type
+	 * @return bool
+	 */
+	public function is_post_type_blacklisted($post_type = '') {
+		if(empty($post_type)) {
+			$post_type = $this->type();
+		}
+		return (bool) in_array($post_type, $this->blacklisted_post_types);
+	}
+
+	public function get_blacklisted_values() {
+		if (!array($this->blacklisted_post_types)) return '';
+
+		return implode(', ', array_map(function($v) { return "'$v'"; }, $this->blacklisted_post_types ));
```

Comment:
Oh, I see what you did here–very neat. So this function returns them formatted for SQL? Are closures php5.3+ compatible?